platform/openide.filesystems/src/org/openide/filesystems/Ordering.java

```diff
@@ -158,7 +164,18 @@ public int compareTo(ChildAndPosition o) {
                 previousChild = subsequentChild;
             }
         }
-        if (logWarnings && /* #201893*/ !parent.getPath().matches("Projects/.+/Lookup") && !childrenByPosition.isEmpty() && childrenByPosition.size() < children.size()) {
+        boolean shouldLog = logWarnings && !childrenByPosition.isEmpty() && childrenByPosition.size() < children.size();
+        if (shouldLog) {
+            // exceptions:  See Bugzilla#201893 
+            if (parent.getPath().matches("Projects/.+/Lookup")) {
+                // do not report any order issues for Projects/Lookup, even between files, or between folders.
+                shouldLog = false;
+            } else if (FileUtil.isParentOf(FileUtil.getConfigRoot(), parent) && ((ordered & unordered) == 0)) {
```

Comment:
both ordered and undordered bits have to be set for a category (folder OR file) to log the warning.